scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -918,6 +924,428 @@ cdef class TransformedDensityRejection(Method):
+        Higher orders result in fewer intervals for the approximations. Default
+        is 3.
+    u_resolution : float, optional
+        Set maximal tolerated u-error. Values of u_resolution must at least 1.e-15 and
+        1.e-5 at most. Notice that the resolution of most uniform random number sources
+        is 2-32= 2.3e-10. Thus a value of 1.e-10 leads to an inversion algorithm that
+        could be called exact. For most simulations slightly bigger values for the
+        maximal error are enough as well. Default is 1e-10.
+    max_intervals : int, optional
+        Set maximum number of intervals. max_ivs must be at least 100 and at most 1000000.
+        Default is 10000.
+    keep_cdf : bool, optional
+        If the PDF is given, then the CDF is computed numerically from the given PDF using
+        adaptive Gauss-Lobatto integration. Thus a table of CDF points is stored to keep
+        the number of evaluations of the PDF minimal. Usually this table is discarded when
+        the setup is completed. If ``keep_cdf`` is TRUE, then this table is kept and the
+        approximated CDF can be evaluated by calling the `cdf` method. Default is ``False``.
+    random_state : {None, int, `numpy.random.Generator`,
+                        `numpy.random.RandomState`}, optional
+
+        A NumPy random number generator or seed for the underlying NumPy random
+        number generator used to generate the stream of uniform random numbers.
+        If `random_state` is None (or `np.random`), the `numpy.random.RandomState`
+        singleton is used.
+        If `random_state` is an int, a new ``RandomState`` instance is used,
+        seeded with `random_state`.
+        If `random_state` is already a ``Generator`` or ``RandomState`` instance then
+        that instance is used.
+
+    References
+    ----------
+    .. [1] G. Derflinger, W. Hörmann, and J. Leydold (2008). Numerical inversion when
+           only the density function is known, Research Report Series of the Department
+           of Statistics and Mathematics 78, WU Wien, Augasse 2-6, A-1090 Wien, Austria,
+           http://epub.wu.ac.at/english/
+    .. [2] UNU.RAN reference manual, Section 5.3.12,
+           "PINV – Polynomial interpolation based INVersion of CDF",
+           https://statmath.wu.ac.at/software/unuran/doc/unuran.html#PINV
+
+    Examples
+    --------
+    >>> from scipy.stats import NumericalInversePolynomial
+    >>> from scipy.stats import norm
+
+    To create a generator to sample from the standard normal distribution, do:
+
+    >>> class StandardNormal:
+    ...    def pdf(self, x):
+    ...        return np.exp(-0.5 * x*x)
+    ... 
+    >>> dist = StandardNormal()
+    >>> urng = np.random.default_rng()
+    >>> rng = NumericalInversePolynomial(dist, random_state=urng)
+
+    Once a generator is created, samples can be drawn from the distribution by calling
+    the `rvs` method:
+
+    >>> rng.rvs()
```

Comment:
I propose to add a histogram for all the UNURAN methods that we implement => it is the most convincing way to show that rvs works :)

Reply:
Done.